client/components/review.js

```diff
@@ -17,23 +17,34 @@ class Review extends React.Component {
 
   render() {
     const { label } = this.props.altLabels ? this.props.alt : this.props;
-    const { hint, isGranted, showGrantedLabel = true, review } = this.props;
+    const {
+      hint,
+      isGranted,
+      showGrantedLabel = true,
+      review,
+      changedFromLatest,
+      changedFromGranted
+    } = this.props;
     return (
       <div className="review">
         {
           (!isGranted || showGrantedLabel) && <h3>{review || label}</h3>
         }
-        <ChangedBadge
-          changedFromLatest={this.props.changedFromLatest}
-          changedFromGranted={this.props.changedFromGranted}
-          protocolId={this.props.protocolId}
-        />
         {
-          this.props.readonly && (this.props.changedFromLatest || this.props.changedFromGranted) && (
+          (changedFromLatest || changedFromGranted) && (
```

Comment:
Should probably do a `return null` in the `ChangedBadge` component too if the props are falsy.

No objection to having this as well as a belt'n'braces, but putting it in the component feels generally safer.